src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -463,6 +466,9 @@ private BoundPattern BindDiscardPattern(DiscardPatternSyntax node, TypeSymbol in
                     SyntaxToken identifier = singleVariableDesignation.Identifier;
                     SourceLocalSymbol localSymbol = this.LookupLocal(identifier);
 
+                    if (!permitDesignations && !identifier.IsMissing)
+                        diagnostics.Add(ErrorCode.ERR_DesignatorBeneathPatternCombinator, identifier.GetLocation());
```

Comment:
I thought we typically put braces even on ifs like these.

Reply:
Some people like to.  I don't.